code/datums/diseases/advance/symptoms/weight.dm

```diff
@@ -1,53 +1,6 @@
 /*
 //////////////////////////////////////
 
-Weight Gain

```

Comment:
Why not port the fat sprite? So being fat makes you look fat.

Reply:
how come